packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [92.0.0]
```

Comment:
@sahar-fehri as you’re introducing a breaking change do you have clients PRs ready ?